docs/architecture/nodes.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Nodes
+authors: Tim McMackin
+last_update:
+  date: 16 April 2024
+---
+
+Tezos nodes are computers that run the Tezos software and participate in the Tezos network.
+Anyone can run a Tezos node.
+
+A Tezos node has three main roles:
+
+- It validates blocks and operations
+- It broadcasts blocks and operations to other nodes and receives them from other nodes over a peer-to-peer network
+- It maintains a copy of the blockchain data and its associated state (also known as the ledger), which includes accounts and their balances, among other things
+
+Beside these technical roles, nodes must satisfy two other important requirements:
+
+- Support the governance of the blockchain
+- Ensure the extensibility of the blockchain with new clients of different kinds
+
+In order to meet these requirements, the software that nodes run is structured according to two major principles:
+
+- It is separated into a protocol and a shell to make it easier to upgrade.
+- It implements a client/server architecture, to allow composition with many other tools in a safe way.
+
+Nodes communicate with clients and with each other through a peer-ro-peer [RPC interface](#the-rpc-interface).
```

Comment:
there is no such thing as a p2p RPC interface: p2p is a network for exchanging data without going through RPCs. Also "cooperate" makes more the distinction between how node *communicate* with each other via p2p, and *cooperate* with each other via RPCs:
Do you mean:
```suggestion
Nodes cooperate with clients and with each other through an [RPC interface](#the-rpc-interface).
```